common/snapshots.py

```diff
@@ -2165,7 +2164,7 @@ def fileInfo(self):
                         continue
                     info = line[:index].strip().split(' ')
                     if len(info) == 3:
-                        d[f] = (int(info[0]), info[1], info[2]) #perms, user, group
+                        d[f.encode('utf-8','replace')] = (int(info[0]), info[1].encode('utf-8','replace'), info[2].encode('utf-8','replace')) #perms, user, group
```

Comment:
I'd rather remove `.decode('utf-8')` in line 2156, change line 2159 to `line.find(b'/')` and line 2166 to `.split(b' ')`. So we can skip all these `.encode(...)` in here